orc8r/cloud/go/obsidian/swagger/v1/index.html

```diff
@@ -36,6 +36,9 @@
 
 <!-- Sidebar navigation -->
 <div class="sidenav">
+  <!-- Collapse Button -->
+  <button onclick="window.location=window.location + '?docExpansion=none' "> Collapse Tags </button>
```

Comment:
Functionality looks great. Let's spend 10 min getting the aesthetics right

Let's do this

- Add `<script src="https://kit.fontawesome.com/85d0a9f517.js" crossorigin="anonymous"></script>` to the head of the html template
- Move the button to *after* all the services
- Style the button like this `<button class="fas fa-compress-alt" style="cursor: pointer; font-size: x-large" onclick="alert('hi')"></button>`

By default this will look something like this

![Screen Shot 2021-03-16 at 11 39 31 PM](https://user-images.githubusercontent.com/8029544/111426175-3ff91d00-86c2-11eb-8ced-a1d29d7caa4a.png)